cmd/namespace.go

```diff
@@ -84,15 +83,12 @@ Please note that at this time all namespaces created in the registry are world-r
 
 			return err
 		},
-		Args:        cobra.RangeArgs(1, 3),
+		Args:        cobra.RangeArgs(1, 1),
```

Comment:
For fixed number of arguments, cobra recommand using `cobra.ExactArgs` instead of `cobra.RangeArgs`.